runtime/src/test/resources/tailcall/runtime/graphql/test-modify-interface-field.graphql

```diff
@@ -0,0 +1,39 @@
+#> server-sdl
+schema {
+  query: Query
+}
+
+interface IA {
+  a: String
+}
+
+type B implements IA {
+  a: String @modify(name: "x")
+  b: String
+}
+
+type Query {
+  bar: B
+}
+
+#> client-sdl
+schema {
+  query: Query
+}
+
+interface IA {
+  a: String
+}
+
+type B implements IA {
+  b: String
+  x: String
+}
+
+type Query {
+  bar: B
+}
```

Comment:
Will the SDL be checked if there is a compilation error?

Reply:
I think we can drop the client-SDL here. Since we are asserting on errors thrown.